app/assets/v2/js/grants/detail.js

```diff
@@ -70,6 +110,7 @@ $(document).ready(function() {
           .send({from: accounts[0], gasPrice: 4000000000})
           .on('transactionHash', function() {
             document.issueURL = document.getElementById('grant-link').href;
+            $('.modal .close').trigger('click');
             enableWaitState('#grants-details');
           })
           .on('confirmation', function(confirmationNumber, receipt) {
```

Comment:
Im worry about how we are doing this if someone decide to reload the page in the waitState we have some kind of mechanist to check if the grants is sync?

Reply:
I think this question is there in all the pages if someone refreshes at this stage. It will show the particular page instead of the waiting state. I think not related to this PR. But I will check on it.

Reply:
yes yes agree is not related at all but yesterday was looking the cancel code and started to think about the closed grants in blockchain but not in our db